.circleci/config.yml

```diff
@@ -121,6 +121,26 @@ workflows:
                 - main
           requires:
             - deploy_dev
+      - request-test-approval:
+          type: approval
+          requires:
+            - deploy_dev
+            - e2e_environment_test
+      - hmpps/deploy_env:
+          name: deploy_test
+          env: 'test'
+          jira_update: true
+          context:
+            - hmpps-common-vars
+            - hmpps-community-accommodation-tier-2-ui-stage
```

Comment:
I'm wondering why this is 'context' is known as 'stage'? Wouldn't 'test' be more consistent and clear?

Reply:
yep it would, that's a good point. and this reminds me that I need to ask how this context was setup in CircleCI, as afaik it doesn't exist for us

Reply:
oh just checked CircleCi and this context does now exist for our project, I think that it's generated by moj team as all the other projects have `-stage` on them, so won't rock the boat at this point 😬